### ISSUE TITLE
feature request: resources limits

### ISSUE THREAD

**Issue by mausch** [nekomeowww/ollama-operator#2]
Hi and thanks for this project!

Is there any way to define resource requirements per model? I can't seem to find it?
It looks like at the moment the deployments are created without any resource requirements / limits.
I'd like to define in the Model CRD that I want e.g. 4 CPUs and 8Gi memory.

Thanks.

**Comment by nekomeowww** [comment]
Currently I haven't implemented the resource limit for `Model` deployments, I will add the support for it ASAP today.

You can try to use 

```shell
kubectl set resources deployment -l model.ollama.ayaka.io=<model name> --limits cpu=4
```

and

```shell
kubectl set resources deployment -l model.ollama.ayaka.io=<model name> --limits memory=8Gi
```

to manually patch it.

- `-l` means select by label
- `--limits` means add annotation for `resources.limits`

Sorry for any of the inconvenient.

**Comment by nekomeowww** [comment]
Implemented and released, please upgrade to [`v0.7.0`](https://github.com/nekomeowww/ollama-operator/releases/tag/v0.7.0) to test it out.

**Comment by mausch** [comment]
I've defined resources as documented in https://ollama-operator.ayaka.io/pages/en/references/crd/model.html#full-crd-reference but the model deployment isn't getting them...
